lang/cs/Org.Apache.REEF.Network/Group/Driver/ICommunicationGroupDriver.cs

```diff
@@ -56,8 +58,9 @@ public interface ICommunicationGroupDriver
         /// <param name="masterTaskId">The master task id in broadcast operator</param>
         /// <param name="codecType">The Codec used for serialization</param>
         /// <param name="topologyType">The topology type for the operator</param>
+        /// <param name="pipelineDataConverter">The class used to convert data back and forth to pipelined one</param>
         /// <returns>The same CommunicationGroupDriver with the added Broadcast operator info</returns>
-        ICommunicationGroupDriver AddBroadcast<T>(string operatorName, string masterTaskId, ICodec<T> codecType, TopologyTypes topologyType = TopologyTypes.Flat);
+        ICommunicationGroupDriver AddBroadcast<T>(string operatorName, string masterTaskId, ICodec<T> codecType, TopologyTypes topologyType = TopologyTypes.Flat, IPipelineDataConverter<T> pipelineDataConverter = null);
```

Comment:
In which case I'd prefer an overloaded method. One which takes the parameters, another which doesn't. Unless that is uncommon in C#...